src/styles/styles.less

```diff
@@ -1,28 +1,44 @@
-@import './ironsworn-icons.less';
-@import 'mixins.less';
+@import 'mixin-utils.less';
+@import 'attr-box.less';
+@import 'button.less';
+@import 'chat-messages.less';
+@import 'compact-sheet.less';
+@import 'dialogs.less';
+@import 'forms.less';
+@import 'icons.less';
+@import 'item-list.less';
+@import 'layout.less';
+@import 'rolls.less';
+@import 'sidebar.less';
+@import 'tables.less';
+@import 'transitions.less';
+@import 'typography.less';
+@import 'widgets.less';
```

Comment:
So this reorg I think is emblematic in how our styles differ. 😁 This PR is an 83-file monster that's pretty hard to review, because there are at least 2 separate mostly-mechanical refactorings mixed in with at least 3 substantive changes to the styles themselves. On its own, I 😍 this **specific** change a lot. But it's _super_ hard to review when a big reorg like this is mixed in with changes to the content that's being moved around – this file has 875 lines of diff, and GitHub didn't even display it by default.

One way to make this easier to manage would be to back out and do specific, targeted refactorings in their own PRs. One PR to split this CSS into multiple files (easy to review and merge), one PR to introduce and roll out a global border variable (easy to review and merge), and so on. Does that make sense?